bungee/src/main/java/codecrafter47/bungeetablistplus/handler/AbstractTabOverlayHandler.java

```diff
@@ -378,8 +382,8 @@ public PacketListenerResult onPlayerListHeaderFooterPacket(PlayerListHeaderFoote
             enterHeaderAndFooterOperationMode(HeaderAndFooterOperationMode.PASS_TROUGH);
         }
 
-        this.serverHeader = packet.getHeader() != null ? packet.getHeader() : EMPTY_JSON_TEXT;
-        this.serverFooter = packet.getFooter() != null ? packet.getFooter() : EMPTY_JSON_TEXT;
+        this.serverHeader = packet.getHeader() != null ? ((TextComponent)packet.getHeader()).getText() : EMPTY_JSON_TEXT;
+        this.serverFooter = packet.getFooter() != null ? ((TextComponent)packet.getFooter()).getText() : EMPTY_JSON_TEXT;
```

Comment:
Shouldn't this be an empty string rather than `EMPTY_JSON_TEXT` when the packet properties are null? Seems more consistent to the `getText()` method call